packages/api/src/controllers/region.js

```diff
@@ -0,0 +1,63 @@
+import Router from 'express/lib/router'
+import { authMiddleware } from '../middleware'
+import { db } from '../store'
+
+const app = Router()
+
+app.get('/', authMiddleware({ admin: true }), async (req, res, next) => {
+  const [regions, cursor] = await db.region.find()
+  if (req.query.grouped) {
+    return res.json(regions)
+  }
+
+  const flatOrchList = []
+  regions.forEach((region) => {
+    region.orchestrators.forEach((orch) => {
+      orch.region = region.region
+      flatOrchList.push(orch)
+    })
+  })
+
+  return res.json(flatOrchList)
+})
+
+app.get('/:region', authMiddleware({ admin: true }), async (req, res, next) => {
+  const region = await db.region.get(req.params.region)
+  return res.json(region)
+})
+
+app.put('/:region', authMiddleware({ admin: true }), async (req, res, next) => {
+  if (!req.body || !req.body.region) {
+    res.status(406)
+    return res.json({ error: 'region field is required' })
+  }
+
+  if (!req.body.orchestrators || !req.body.orchestrators.length) {
```

Comment:
An absent `req.body.orchestrators` will be treated as an error condition by the `handlePost` middleware. But an empty list for a region is totally legit, don't necessarily need to log it